ui/app/components/clients/attribution.js

```diff
@@ -103,9 +104,24 @@ export default class Attribution extends Component {
     }
   }
 
-  get getCsvData() {
+  destructureCountsToArray(object) {
+    let { clients, entity_clients, non_entity_clients } = object;
```

Comment:
are clients, entity_clients, etc. here are numbers?  e.g. clients = 100? or are they returning objects. It maybe helpful for a future developer if making changes to understand what is expected. So maybe just a comment explaining the type returned. Because we don't use something like typescript, it's not always easy to understand what is expected.

Reply:
great idea! thank you